src/System/refine.rs

```diff
@@ -912,23 +481,18 @@ fn check_preconditions(
         }
     }
 
-    return true;
+    true
 }
 
 pub fn find_extra_input_output(
```

Comment:
In the future we should probably move this out of refine too as it is only used in main.rs when we aren't doing refinement.